### PR TITLE
Add translation to HowDoesItWork

### PR DESCRIPTION
**Pull request by khaldoon9kh**

### CODE REVIEW

**Comment by khaldoon9kh** [comment]
Added Translation to both: "How Does it Work" and "trustedBy" components

**Review by fadime-ozdemir — APPROVED** [review]
LGTM

**Review by fadime-ozdemir — APPROVED** [review]
LGTM

**Review by fadime-ozdemir — CHANGES_REQUESTED** [review]
tests aren't passing

**Comment by khaldoon9kh** [comment]
PTAL

File: src/locales/en.json [hunk 1]
```diff
@@ -67,6 +67,13 @@
       "blogLink": "Go to blog post",
       "buttonSeeMore": "See more blogs"
     },
+    "howDoesItWork": {
+      "how": "How Does It Work?",
+      "first": "Select Your Preference",
```
Comment (halitbatur) [inline src/locales/en.json]: the key name could be better maybe smth like preference service and connect

**Review by halitbatur — CHANGES_REQUESTED** [review]

File: src/locales/en.json (hunk 1 above)
Comment (khaldoon9kh) [reply src/locales/en.json]: Revised

**Comment by khaldoon9kh** [comment]
PTAL

**Review by halitbatur — APPROVED** [review]
resolve conflicts and its ready to be merged

**Pull request merged**